ptu/global.ini

```diff
@@ -69620,10 +69620,10 @@ vehicle_room_cargo_bay=Frachtraum
 vehicle_room_cockpit=Cockpit
 vehicle_room_component_access=Komponentenzugriff
 vehicle_room_crew=Mannschaftsunterkünfte
-vehicle_room_engineering=Technik
+vehicle_room_engineering=Technikraum
 vehicle_room_foyer=Foyer
 vehicle_room_habitation=Wohnraum
-vehicle_room_locker=Schließfach
+vehicle_room_locker=Umkleideraum
```

Comment:
vehicle_room_locker=Locker
Hm. Ist mMn einfach nur das Schließfach gemeint

Reply:
Ich würde da gar nicht auf die variable schauen sondern die englische Datei vergleichen. Mach ich zumindest so. In Englisch steht da einfach nur "Locker"